docs/ql-libraries/dataflow/dataflow.md

```diff
@@ -186,6 +186,45 @@ values, for example through `out` parameters in C#, the `ReturnKind` class can
 be defined and used to match up different kinds of `ReturnNode`s with the
 corresponding `OutNode`s.
 
+#### Lambda flow
+
+For lambda-like calls, the library supports built-in call resolution based on data flow between a lambda creation expression and a lambda call. The interface that needs to be implemented is
```

Comment:
I'd like it to say first-class function rather than lambda as that's the general case we support. Lambdas can then be mentioned as the common case.
```suggestion
For calls to first-class functions, the library supports built-in call resolution based on data flow between a function creation expression and a call. The interface that needs to be implemented is
```

Reply:
C/C++ is possibly going to want this for function pointers.